go.mod

```diff
@@ -220,5 +220,5 @@ require (
 
 replace github.com/ethereum/go-ethereum v1.13.8 => github.com/ethereum-optimism/op-geth v1.101308.1-rc.1
 
-//replace github.com/ethereum-optimism/superchain-registry/superchain => ../superchain-registry/superchain
+replace github.com/ethereum-optimism/superchain-registry/superchain => ../superchain-registry/superchain
```

Comment:
This temporary change needs to be undone before merging. And a newer version of the registry imported (doesn't exist yet).

Reply:
you can just reference the commit of your superchain-registry pr in this pr to test everything together already.
```
go get github.com/ethereum-optimism/superchain-registry/superchain@177cc47729e8d169581fc00b928a3b02c7ccde98
```